Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.3.5-alpine3.20
+FROM ruby:3.3.5-alpine3.21
```

Comment:
We've had to roll out ruby 3.3.8 (couldn't find alpine3.21 for 3.3.5) on content-platform-forms and contentful-reporting-api so I wonder if we should do this here as well but I'm not familiar with the repo.